test/plugins/cookie/cookies.get.test.js

```diff
@@ -0,0 +1,59 @@
+import sinon from 'sinon';
+
+import CookiesPlugin from '../../../src/plugins/cookies';
+import state from '../data/cookie-state.json';
+import { create } from '../../../src/chrome-api';
```

Comment:
And `create` is too general name. It will be better if you import a module instead of only one function:
```js
import * as chromeApi from '../../../src/chrome-api';
// ...
const chrome = chromeApi.create();
```